packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -281,13 +284,14 @@ export class IndexedDbPersistence implements Persistence {
       this.indexManager,
       /*keepDocumentChangeLog=*/ this.allowTabSynchronization
     );
+    this.window = platform.window;
     if (platform.window && platform.window.localStorage) {
-      this.window = platform.window;
-      this.webStorage = this.window.localStorage;
+      this.webStorage = platform.window.localStorage;
     } else {
-      throw new FirestoreError(
-        Code.UNIMPLEMENTED,
-        'IndexedDB persistence is only available on platforms that support LocalStorage.'
+      this.webStorage = null;
+      log.error(
```

Comment:
I think maybe we should only log the error if force is false, because if force==true I don't think it matters that localStorage is not available.